cli/dcoscli/experimental/main.py

```diff
@@ -71,17 +72,22 @@ def _info():
     return 0
 
 
-def _add(dcos_package):
+def _add(dcos_package, package_name, package_version):
     """
     Adds a DC/OS package to DC/OS
 
     :param dcos_package: path to the DC/OS package
-    :type dcos_package: str
+    :type dcos_package: None | str
+    :param package_name: the name of a remote DC/OS package
+    :type package_name: None | str
+    :param package_version: the version of a remote DC/OS package
+    :type package_version: None | str
     :return: process status
     :rtype: int
     """
     package_manager = get_package_manager()
-    response = package_manager.package_add(dcos_package)
+    response = package_manager.package_add(
+        dcos_package, package_name, package_version)
```

Comment:
I think @larioj is saying that `docopt` guarantee that both are not specified because of the docopt specification so we don't need to do it, no? If both are specified then it means either a bug in the command line specification or in docopt.